test/projects/RteTestM3/RteTestM3.cprj

```diff
@@ -31,7 +31,7 @@
   </compilers>
 
   <target Bname="RteTest Dummy board" Bvendor="Keil" Bversion="1.2.3" Ddsp="NO_DSP" Dendian="Little-endian" Dfpu="NO_FPU" Dmve="NO_MVE" Dname="RteTest_ARMCM3" Dsecure="Non-secure" Dtz="NO_TZ" Dvendor="ARM:82">
-    <output intdir="./" name="RteTestM3" outdir="./" type="exe"/>
+    <output intdir="./" name="RteTestM3" outdir="./" rtedir="CONFIG_FOLDER" type="exe"/>
```

Comment:
This test case is used in several places. It should not be modified.
Instead  an additional project should be used that specifies an RTE directory.
The corresponding config files should not be renamed either